<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSBizLivePusher</class>
 <widget class="QWidget" name="GBSBizLivePusher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="dwgtDanmaku" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>-40</y>
     <width>671</width>
     <height>961</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="lblPreview">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>637</width>
      <height>468</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">      background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, 
           stop:0 #000000, stop:0.525 #434343, stop:1 #000000); 
     border-radius: 20px;
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>453</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 16px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="lblCamera">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>32</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>摄像头驱动</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMicrophone">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>86</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>麦克风驱动</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxCamera">
     <property name="geometry">
      <rect>
       <x>113</x>
       <y>27</y>
       <width>303</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxMicrophone">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>81</y>
       <width>303</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="butStartLive">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>550</y>
      <width>150</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:Resources/gbs-start-living.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>710</y>
      <width>631</width>
      <height>106</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F9F9F9;
border: 1px solid  white;
border-radius: 16px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>52</y>
       <width>313</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>52</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxVendor">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>52</y>
       <width>109</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>22</y>
       <width>158</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>添加直播间地址（转播）</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>840</y>
      <width>631</width>
      <height>106</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F9F9F9;
border: 1px solid  #FFFFFF;
border-radius: 16px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>违规提醒</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>559</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#00C566;</string>
     </property>
     <property name="text">
      <string>使用说明</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAgainstRule">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-image: url(:Resources/gbs-widget-docking.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>525</width>
     <height>921</height>
    </rect>
   </property>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="btnF2F">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>留言互动</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnShowDanmaku">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTips">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>礼物打赏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFans">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>谁来了</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPrompter">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDockable">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>521</width>
       <height>821</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
